README.md

```diff
@@ -1,18 +1,46 @@
-A development day project for Suzanne to refresh her Scala knowledge.
 
-# Run Github monitor
+# Pull Request Monitor
+This project started as a development day project for Suzanne to refresh her Scala knowledge.  
+It supports the monitoring of pull requests in repositories that have specific GitHub team access.  
+It will send slack alerts to the channel associated with the SLACK_WEBHOOK
```

Comment:
Maybe add a bit to this sentence explaining what the bullet points are, ie the format of the Slack alert.